crypto/README.md

```diff
@@ -1,4 +1,12 @@
-Crypto
+Tezos crypto
 ===========
 
-Component contains cryptographic algorithms for encryption and decryption of messages
\ No newline at end of file
+Component contains cryptographic algorithms for hashing, signing & signature verification, with a slant towards
+those used by `tezos` - and in particular the `Kernel SDK` for smart rollups.
```

Comment:
If possible. are there links to docs for tezos and kernel sdk that we could add?

Reply:
have done for `tezos` - but for now there's nothing really linkable for the SDK - will update once we have some proper SDK docs